mne/source_space.py

```diff
@@ -2488,7 +2536,7 @@ def get_volume_labels_from_src(src, subject, subjects_dir):
         roi_str = src[nr]['seg_name']
         try:
             ind = all_labels_aseg[0].index(roi_str)
-            color = np.array(all_labels_aseg[1][ind]) / 255
+            color = tuple(np.array(all_labels_aseg[1][ind]) / 255.)
```

Comment:
why `tuple` this?

Reply:
I did this change a lot of months ago so now I don't remember the reason...
Can I leave tuple?

Reply:
ok, I reverted it!
If I remember the reason I'll update you.